home/templates/home/base.html

```diff
@@ -10,6 +10,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% site_title %}</title>
 
+ 
+
+
+
+
```

Comment:
There are still many empty lines, like here.

I know it's nit-picky, but we have to try to keep the code as consistent and readable as possible